### PR TITLE
NATIVE-524: Hide news and extras if disabled

### PR DESCRIPTION
**Pull request by Schedulaar**

### CODE REVIEW

File: src/modules/endpoint/sagas/loadEvents.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import determineApiUrl from '../determineApiUrl'
 function * loadEvents (
   city: string,
   language: string,
+  eventsEnabled: boolean,
```
Comment (klinzo) [inline src/modules/endpoint/sagas/loadEvents.js:12]: Maybe we should switch to an object as param at some point.

File: src/modules/endpoint/sagas/loadCityContent.js [hunk 2]
```diff
@@ -81,7 +83,7 @@ export default function * loadCityContent (
 
   const [categoriesMap, events] = yield all([
     call(loadCategories, newCity, newLanguage, dataContainer, shouldUpdate),
-    call(loadEvents, newCity, newLanguage, dataContainer, shouldUpdate)
+    call(loadEvents, newCity, newLanguage, cityModel.eventsEnabled, dataContainer, shouldUpdate)
```
Comment (klinzo) [inline src/modules/endpoint/sagas/loadCityContent.js:86]: Why do we have to set events to an empty array if events are disabled? Can't we just return an empty array here if disabled?

Comment (Schedulaar) [reply src/modules/endpoint/sagas/loadCityContent.js:86]: The data container has to be set as well, since other sagas might as well query events from the data container (and should receive an empty array).

Comment (klinzo) [reply src/modules/endpoint/sagas/loadCityContent.js:86]: Might query or are querying?

Comment (Schedulaar) [reply src/modules/endpoint/sagas/loadCityContent.js:86]: Are querying

File: src/modules/endpoint/sagas/loadEvents.js (hunk 1 above)
Comment (Schedulaar) [reply src/modules/endpoint/sagas/loadEvents.js:12]: Yeah, but I don't want to bloat this PR.

**Comment by Schedulaar** [comment]
As these are changes to the core sagas, I'd also explicitly request a review from @maxammann.

File: src/modules/endpoint/sagas/__mocks__/loadCities.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ function * loadCities (
 ): Saga<Array<CityModel>> {
   const citiesAvailable = yield call(() => dataContainer.citiesAvailable())
 
-  if (!citiesAvailable || forceRefresh) {
+  if (!citiesAvailable) {
```
Comment (maxammann) [inline src/modules/endpoint/sagas/__mocks__/loadCities.js:14]: Why was this necessary? this should behave in the same way as the non-mock loadCities.js.

`forceRefresh` should probably not be true in the tests.

Comment (maxammann) [reply src/modules/endpoint/sagas/__mocks__/loadCities.js:14]: This is because the tests should not depend on refreshing data and calling the DataContainer.

Comment (Schedulaar) [reply src/modules/endpoint/sagas/__mocks__/loadCities.js:14]: Because it is necessary to also test, that if the lastUpdate is quite a while ago, `loadCityContent` would call loadCities with `forceRefresh===true`.
(Just like `loadCategories` and  `loadEvents` and `loadLanguages` btw.)
